examples/fluids/index.md

```diff
@@ -634,19 +634,37 @@ The `STGRand.dat` file is the table of the random number set, $\{\bm{\sigma}^n,
 numerous terms in the STG formulation.
 
-| Math            | Label  | $f(\bm{x})$? | $f(n)$? |
-|-----------------|--------|--------------|---------|
-| $ \{\bm{\sigma}^n, \bm{d}^n, \phi^n\}_{n=1}^N$        | RN Set | No           | Yes     |
-| $\bm{\overline{u}}$ | ubar | Yes | No |
-| $U_0$           | U0     | No           | No      |
-| $l_t$           | l_t    | Yes          | No   |
-| $\varepsilon$   | eps    | Yes          | No   |
-| $\bm{R}$        | R_ij   | Yes          | No      |
-| $\bm{C}$        | C_ij   | Yes          | No      |
-| $q^n$           | q^n    | Yes           | Yes     |
-| $\{\kappa^n\}_{n=1}^N$ | k^n  | No           | Yes      |
-| $h_i$           | h_i    | Yes          | No   |
-| $d_w$           | d_w    | Yes          | No   |
+| Math                                           | Label    | $f(\bm{x})$?   | $f(n)$?   |
+| -----------------                              | -------- | -------------- | --------- |
+| $ \{\bm{\sigma}^n, \bm{d}^n, \phi^n\}_{n=1}^N$ | RN Set   | No             | Yes       |
+| $\bm{\overline{u}}$                            | ubar     | Yes            | No        |
+| $U_0$                                          | U0       | No             | No        |
+| $l_t$                                          | l_t      | Yes            | No        |
+| $\varepsilon$                                  | eps      | Yes            | No        |
+| $\bm{R}$                                       | R_ij     | Yes            | No        |
+| $\bm{C}$                                       | C_ij     | Yes            | No        |
+| $q^n$                                          | q^n      | Yes            | Yes       |
+| $\{\kappa^n\}_{n=1}^N$                         | k^n      | No             | Yes       |
+| $h_i$                                          | h_i      | Yes            | No        |
+| $d_w$                                          | d_w      | Yes            | No        |
+
+#### Internal Damping Layer (IDL)
+STG creates spurious acoustic waves during its development to natural
+turbulent structures. To damp those out, we use an internal damping layer (IDL)
+to prevent them from growing. This implementation was inspired from
+{cite}`shurSTG2014`, but is implemented here as a ramped volumetric forcing
+term in the residual itself. It takes the following form:
+
+$$
+S(\bm{q}) = -\sigma(\bm{x})\left.\frac{\partial \bm{q}}{\partial \bm{Y}}\right\rvert_{\bm{q}} \bm{Y}'
+$$
+
+where $\bm{Y}' = [P - P_\mathrm{ref}, \bm{0}, 0]^T$, and $\sigma(\bm{x})$ is a
+linear ramp starting at `-idl_start` with length `-idl_length` and an amplitude
+of inverse `-idl_decay_rate`. The damping is defined in pressure-primitive
+variables, but is converted to conservative variables using $\partial
+\bm{q}/\partial \bm{Y}\rvert_{\bm{q}}$, which is linearized about the current
+flow state. $P_\mathrm{ref}$ is defined via the `-refernce_pressure` flag.
```

Comment:
Here's an adjustment. I think Shur's IDL is quite different so it at least deserves a citation to the book chapter or one of the mentioned papers.
```suggestion
The STG inflow boundary condition creates large amplitude acoustic waves.
We use an internal damping layer (IDL) to damp them out without disrupting the synthetic structures developing into natural turbulent structures. This implementation was inspired from
{cite}`shurSTG2014`, but is implemented here as a ramped volumetric forcing
term in the residual itself. It takes the following form:

$$
S(\bm{q}) = -\sigma(\bm{x})\left.\frac{\partial \bm{q}}{\partial \bm{Y}}\right\rvert_{\bm{q}} \bm{Y}'
$$

where $\bm{Y}' = [P - P_\mathrm{ref}, \bm{0}, 0]^T$, and $\sigma(\bm{x})$ is a
linear ramp starting at `-idl_start` with length `-idl_length` and an amplitude
of inverse `-idl_decay_rate`. The damping is defined in terms of a pressure-primitive
anomaly $\bm Y'$ converted to conservative source using $\partial
\bm{q}/\partial \bm{Y}\rvert_{\bm{q}}$, which is linearized about the current
flow state. $P_\mathrm{ref}$ is defined via the `-reference_pressure` flag.
```

Reply:
Done.